### PR TITLE
clean up random lathe recipes

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
everything else
no user facing changes 

requires #31515

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
no cl no fun

### CODE REVIEW

**Comment by slarticodefast** [comment]
When you fix the merge conflict, give me a ping and I will review this. Should be quick and easy.

**Comment by deltanedas** [comment]
@slarticodefast ping

File: Resources/Prototypes/Recipes/Lathes/powercells.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@
     Steel: 500
     Glass: 500
     Plastic: 250
-    Gold: 40
+    Gold: 50
```
Comment (Aeshus) [inline Resources/Prototypes/Recipes/Lathes/powercells.yml:59]: Was this meant to be changed?

Comment (slarticodefast) [reply Resources/Prototypes/Recipes/Lathes/powercells.yml]: I changed it back.

**Review by slarticodefast — APPROVED** [review]
Looks good to me, thanks for the cleanup!

**Review by VasilisThePikachu — APPROVED** [review]

**Pull request merged**